party.supertux.supertuxparty.metainfo.xml

```diff
@@ -0,0 +1,93 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>party.supertux.supertuxparty</id>
+
+  <name>Super Tux Party</name>
+  <summary>A party game starring FLOSS mascots</summary>
+
+  <metadata_license>CC-BY-4.0</metadata_license>
+  <project_license>GPL-3.0-or-later</project_license>
+
+  <recommends>
+	<control>pointing</control>
+	<control>keyboard</control>
+	<control>gamepad</control>
+  </recommends>
+
+  <categories>
+	<category>Game</category>
+	<category>BoardGame</category>
+  </categories>
+
+  <description>
+	<p>
+	  A free/libre and open-source party game that is meant to replicate the feel of games such as Mario Party
+	</p>
+  </description>
+
+  <launchable type="desktop-id">party.supertux.supertuxparty.desktop</launchable>
+
+  <content_rating type="oars-1.0" />
+  <url type="homepage">
+	https://supertux.party
+  </url>
+  <url type="bugtracker">
+	https://gitlab.com/SuperTuxParty/SuperTuxParty/issues/
+  </url>
+  <url type="translate">
+	https://hosted.weblate.org/projects/super-tux-party
+  </url>
+
+  <screenshots>
+	<screenshot type="default">
+	  <image> http://supertux.party/screenshots/1.png </image>
+	</screenshot>
+	<screenshot type="default">
+	  <image> http://supertux.party/screenshots/2.png </image>
```

Comment:
This screenshot is too small.

Reply:
Screenshot is updated with a higher res version